twitterpuller/src/test/resources/application.properties

```diff
@@ -0,0 +1 @@
+kafka.bootstrap-servers
```

Comment:
Why is this added in this PR?

Reply:
I'ts from another Test Case I was working on. Its wrong on this commit.